lib/tx_build/claim_predicates.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Stellar.TxBuild.ClaimPredicates do
+  @moduledoc """
+  `ClaimPredicates` struct definition.
+  """
+  alias Stellar.TxBuild.ClaimPredicate
+  alias StellarBase.XDR.ClaimPredicates
+
+  @behaviour Stellar.TxBuild.XDR
+
+  @type validation :: {:ok, any()} | {:error, error()}
+  @type predicates :: ClaimPredicate.t() | list(ClaimPredicate.t())
```

Comment:
It seems only a `list` of `ClaimPredicates` is valid in the definition of this type.